SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec

```diff
@@ -0,0 +1,226 @@
+# This spec file assumes you're building on an environment where:
+# * You have access to the internet during the build
+# * You have rustup installed on your system
+# * You have both x86_64-unknown-linux-gnu and x86_64-unknown-linux-musl, or
+# * You have both aarch64-unknown-linux-gnu and aarch64-unknown-linux-musl
+#   targets installed.
+
+%define using_rustup 0
+%define using_musl_libc 0
+%define using_vendored_crates 1
+%define using_vendored_crates 1
+
+# Generated by update script
+%global gitver   38.0
+%global gitnum   72
+%global gitrev   gdff4252b
+%global clh_name cloud-hypervisor
+%global clh_type cvm
+
+Name:           %{clh_name}-%{clh_type}
+Summary:        Cloud Hypervisor CVM is an open source Virtual Machine Monitor (VMM) that enables running SEV SNP enabled VMs on top of MSHV using the IGVM file format as payload.
+Version:        %{gitver}.%{gitnum}
+Release:        1000.%{gitrev}%{?dist}
+License:        ASL 2.0 or BSD-3-clause
+Group:          Applications/System
+URL:            https://github.com/cloud-hypervisor/cloud-hypervisor
+Source0:        %{clh_name}-%{version}.tar.gz
+%if 0%{?using_vendored_crates}
+Source1:        %{clh_name}-%{version}-vendor.tar.gz
+Source2:        config.toml
+%endif
+ExclusiveArch:  x86_64
+
+BuildRequires:  gcc
+BuildRequires:  glibc-devel
+BuildRequires:  binutils
+BuildRequires:  git
+BuildRequires:  openssl-devel
+
+%if ! 0%{?using_rustup}
+BuildRequires:  rust >= 1.62.0
+BuildRequires:  cargo >= 1.62.0
+%endif
+
+Requires: bash
+Requires: glibc
+Requires: libgcc
+Requires: libcap
```

Comment:
I think we should add a Conflicts with cloud-hypervisor.

We should then install cloud-hypervisor-cvm to the same places where cloud-hypervisor gets installed to.

Also, please consider changes we may need to make to the kata-containers and/or kata-containers-cvm package sources or SPEC files. We will need to change the paths to cloud-hypervisor.